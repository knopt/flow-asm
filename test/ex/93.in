12
4
9.95678094781653
-0.3566016891681816 -1.9801274535026252 -2.3519310194252663 -1.7448404693790849 8.728346463379161 -1.426025857385671 -4.538071321086443 -5.341856784941825 -6.627041918311916 2.179191213223511 1.7695917050917291 2.3295089620611336 
-8.578225628968447 -6.7721782276367986 4.200195275904887 3.3493515996104684 7.453759260640126 -1.6421291920006453 -4.9324044784746395 -2.690428139211585 3.8741821645863066 -3.192270164972724 2.896111947786931 2.7110000398723564 
-8.626591876363843 -2.643486252990181 9.046200880546824 -3.229836302882454 -7.228971433435101 9.214399664305258 -3.432031547834291 -5.579530407301794 1.9805739748470348 0.9137266657504632 -5.210886644503114 1.02670785097596 
7.358130591425937 -9.388361663263868 7.542698961867835 -9.138466492183518 -1.7775060855940321 -6.86758345068572 -7.945637927604525 -7.092907329206759 -4.454173505928106 -5.304458953290725 3.3562744242497757 -2.0048165894204057 

-7.054272529765329 -1.5860763501365724 -2.924096520346633 -0.6121973780432022 
