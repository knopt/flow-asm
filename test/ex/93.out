-161.146361593641 -99.22425510905785 22.576688719754717 102.12566508808658 -161.79914080616706 185.94121672205114 51.35585620891384 33.13778015607916 149.92365388405577 -122.10835375398523 -32.33977215986453 6.194500394077833 
213.89100173685762 109.4817846178304 -251.71395162755434 -104.45241336769398 -364.87753970040296 246.80100293476798 222.46219309756847 -105.95970920442241 -370.79146071059284 178.83742911056638 -176.53709821145986 -104.26431969607303 
357.68829879695926 -129.98867881069836 -511.61514564005824 306.9155438519859 319.37494180633126 -549.7038157358504 46.22046712808065 12.38491807877615 -255.35966911024556 -115.22993074188602 240.95874483265345 -32.685519427493595 
-333.5349422138774 232.0948152422263 -247.48773851609027 396.84393002379903 -143.80855657005762 200.33972398201178 218.5880397158711 35.9355416302271 22.136970801592867 137.6102361539692 -192.49800975555638 49.63647982416736 
